Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN cd ./backend && go build -o ./headlamp-server ./cmd/
 
 # Keep npm install separated so source changes don't trigger install
 FROM base-build AS frontendinstall
+
+# We need .git in order to get the git version for the frontend/.env file
+COPY ./.git /headlamp/.git
```

Comment:
Seems a lot for just the version.

Reply:
I tried copying just a few dirs/files, but git can only fully work if it's got some minimum structure, and it's not possible unless I copy everything AFAIU.
I could run the version command from outside and pass the version to the Dockerfile, but then it's dependent on running the Dockerfile with an external command, not sure it's great for orthogonality.

Reply:
I guess it won't slow down the workflow so much. My .git is only 64MB so far, and only takes 1s to copy with a cold cache/0.2s for a warm file cache. 

We can address this later if needed IMHO.